crd/overlayextensionconfig/api/v1alpha/groupversion_info.go

```diff
@@ -0,0 +1,23 @@
+//go:build !ignore_uncovered
+// +build !ignore_uncovered
+
+// Package v1alpha contains API Schema definitions for the acn v1alpha API group
+// +kubebuilder:object:generate=true
+// +groupName=acn.azure.com
+package v1alpha
```

Comment:
let's do v1alpha1, it was a mistake in NNC api version to leave off the trailing 1

Reply:
Changed to v1alpha1